backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java

```diff
@@ -143,4 +150,22 @@ public UUID getPriorCorrectedTestEventId() {
   public DeviceSpecimenType getDeviceSpecimenType() {
     return order.getDeviceSpecimen();
   }
+
+  // This logic (specifically, the findAny) will need to be updated later on in the multiplex
+  // process - this method is temporary
+  // Eventually, this method will be deprecated in favor of getResultSet()
+  public TestResult getTestResult() {
```

Comment:
This is assuming there will only be one covid result? should we filter on the covid one and keep this to be only for covid as it gets deprecated

Reply:
That is what I'd prefer! But you can't inject a `repository` into an Entity class, Hibernate throws a fit. And we can't get the covid supported disease out of the database without the repository. This is a temporary shim so we can keep the `TestResult getResult()` method signature the same as it used to be, and minimize changes for this PR (it was previously defined in `BaseTestInfo`). In the future we'll need to switch to an explicit `getResultForDisease` call in the Service layer.